tests/conftest.py

```diff
@@ -47,6 +48,50 @@ def rocket(solid_motor):
     return example_rocket
 
 
+@pytest.fixture
+def m():
+    return numericalunits.m
+
+
+@pytest.fixture
+def kg():
+    return numericalunits.kg
+
+
+@pytest.fixture
+def dimensioneless_solid_motor(kg, m):
+    example_motor = SolidMotor(
+        thrustSource="data/motors/Cesaroni_M1670.eng",
+        burnOut=3.9,
```

Comment:
Should the burnOut time be given in seconds?

```suggestion
def dimensioneless_solid_motor(kg, m, s):
    example_motor = SolidMotor(
        thrustSource="data/motors/Cesaroni_M1670.eng",
        burnOut=3.9*s,
```